### PR TITLE
Update-prowlarr

### PR DESCRIPTION
**Pull request by RaneyDazed**
# Please describe the purpose of this Pull Request.

Update the prowlarr page with more relevant info and help on setting it up. Provided 2 screenshots, but not sure if the second one is necessary? Just thought it would be helpful for people looking for an alternative to nzbhydra2 and jackett.

### CODE REVIEW

File: docs/apps/prowlarr.md [hunk 1]
```diff
@@ -22,4 +25,35 @@ sb install prowlarr
...
 
+***Adding Indexers to Sonarr/Radarr via Prowlarr***
+
+#### Under **Settings >> Apps**
+
+1. Click the big `+` sign to add an App.
+2. Provide a "Name", ie Sonarranime if you have multiple instances.
+![Prowlarr Apps](../images/prowlarr-apps.png)
+3. Provide the url for Prowlarr, this should be `http://prowlarr:9696`
+4. Provide the *local* url for [Sonarr](../apps/sonarr.md)/[Radarr](../apps/radarr.md), ie `http://sonarr:8989` or `http://radarr:7878`
```
Comment (maximuskowalski) [inline docs/apps/prowlarr.md:36]: For consistency and the literratti you should probably follow through with the sonarranime in examples shown

- "2. Provide a "Name", ie Sonarranime if you have multiple instances."
- "4. Provide the *local* url for [Sonarr](../apps/sonarr.md)/[Radarr](../apps/radarr.md), ie `http://sonarr:8989` or `http://radarr:7878`" 

So I would suggest maybe including  `http://sonarranime:8989` in the examples shown in "4"

Comment (RaneyDazed) [reply docs/apps/prowlarr.md:36]: I'll try and get to this tomorrow max! sorry. New class been taking all my time. And the kids starting baseball :p

File: docs/apps/prowlarr.md [hunk 2]
```diff
@@ -22,4 +25,35 @@ sb install prowlarr
...
+#### Other
+
+Its probably a good idea to also install **[Flaresolverr](../sandbox/apps/flaresolverr.md)**
+
+Some indexers won't work without it.
+
+To use Prowlarr for torznab feeds, you will need to click on the **Indexer Name** and find the **Torznab URL**.
+![Prowlarr Indexers](../images/prowlarr-indexers.png)
+
+Depending on your usecase, you will need to convert it to a local url, ie `http://prowlarr:9696/2/api`
```
[the remark this answers is not in the document]
Comment (RaneyDazed) [reply docs/apps/prowlarr.md:55]: Probably, I considered that but thought I'd commit as is and see what you thought.

**Comment by maximuskowalski** [comment]
Are you still planning to add http://sonarranime:8989 to the examples here?

**Comment by RaneyDazed** [comment]
Yes I am! Just working on a big project for my college course rn max, sorry. On Aug 24, 2023, at 2:45 AM, Max Kowalski ***@***.***> wrote:﻿
Are you still planning to add http://sonarranime:8989 to the examples here?

**Comment by saltydk** [comment]
I'm closing this due to lack of progress. Reopen a PR for traefik3 if you are still working on this.

**Pull request closed without merging**